sklearn/utils/tests/test_clibs.py

```diff
@@ -0,0 +1,114 @@
+import os
+
+import pytest
+
+from sklearn.utils.testing import SkipTest
+from sklearn.utils._clibs import (get_thread_limits, set_thread_limits,
+                                  get_openblas_version, thread_limits_context,
+                                  _CLibsWrapper)
+
+
+SKIP_OPENBLAS = get_openblas_version() is None
```

Comment:
should not be openBlas also tested? meaning install it in CI if needed...

Reply:
It's tested, and we do install it in at least 1 CI job. However, some versions of OpenBLAS have a bug when displaying the number of threads. We need to skip the tests for these versions